File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        n = len(s)
        countOnes = s.count('1')
        countZeros = n - countOnes
        return "1" * (countOnes-1) + ("0" * countZeros) + "1"
        

File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.cpp
class Solution {
public:
    string maximumOddBinaryNumber(string s) {
        int n = s.length();
        int countOnes = 0;
        for(int i=0; i<n; i++){
            if(s[i] == '1'){
                countOnes++;
            };
        };
        int countZeros = n - countOnes;
        string result = "";
        for(int i=0; i<countOnes-1; i++){
            result.push_back('1');
        };
        for(int i=0; i<countZeros; i++){
            result.push_back('0');
        };
        result.push_back('1');
        return result;
    }
};